src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -177,11 +178,11 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, ...tokenAppro
           <br />
           <br />
           For more details around the token, please read{' '}
-          <a href="https://cow.fi" target="_blank" rel="noreferrer">
+          <a href={COW_LINKS.vCowPost} target="_blank" rel="noreferrer">
             the blog post
           </a>
           .<br /> For more details about the claiming process, please read the{' '}
-          <a href="https://cow.fi" target="_blank" rel="noreferrer">
+          <a href={COW_LINKS.stepGuide} target="_blank" rel="noreferrer">
```

Comment:
not from this PR, but should be this a `ExternalLink` so we track it's click in analytics?

Reply:
@anxolin Wasn't aware that we attached GA events to the ExternalLink, but inspected the component now and that's great. I think this PR is small so I will refactor already here.

Reply:
Now pushed, thank you for the suggestion.

Reply:
nice!